src/CoffeeMatch.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+
+const CoffeeMatch = () => {
+  [
+    {
+      name: "aynu",
+      image: "./assets/aynu.png",
+      qr: 88,
+    },
+    {
+      name: "beza",
+      image: "./assets/beza.png",
+      rating: 92,
+    },
+    {
+      name: "desta",
+      image: "./assets/desta.png",
+      rating: 87,
+    },
+    {
+      name: "kafa",
+      image: "./assets/kafa.png",
+      rating: 89,
+    },
+    {
+      name: "lomi",
+      image: "./assets/lomi.png",
+      rating: 88,
+    },
+    {
+      name: "maaza",
+      image: "./assets/maaza.png",
+      rating: 5,
+    },
+    {
+      name: "nehi",
+      image: "./assets/nehi.png",
+      rating: 87,
+    },
+    {
+      name: "saba",
+      image: "./assets/saba.png",
+      rating: 87,
+    },
+    {
+      name: "worke",
+      image: "./assets/worke.png",
+      rating: 88,
+    },
+    {
+      name: "zala",
+      image: "./assets/zala.png",
+      rating: 87,
+    },
+    {
+      name: "zenu",
+      image: "./assets/zenu.png",
+      rating: 82,
+    },
+  ];
```

Comment:
Not sure what this is doing here. Should this be a prop?

Reply:
I moved the above code to app.js and I used name, image and qr as a props in coffeeMatch.js